### DISCUSSION TITLE
Add notes before the gpx import

### DISCUSSION THREAD

**Discussion by bobblkabb** [organicmaps/organicmaps#8467] [Q&A] [answered]
I have exported gpx data from an online service. I would like to add information to these gpx so that when I import them into Organic Maps, descriptions are included.

Is there an online service for this? Or if necessary a Linux app (openSUSE)?

**Comment by biodranik** [comment, marked as answer]
You may try https://umap.openstreetmap.fr/en/

**Reply by bobblkabb** [reply]
That's exactly what I've been looking for. Thanks a lot!
